tools/ci_build/build.py

```diff
@@ -388,8 +388,7 @@ def parse_arguments():
         "--use_vstest", action='store_true',
         help="Use use_vstest for running unitests.")
     parser.add_argument(
-        "--use_mimalloc", default=['none'],
-        choices=['none', 'stl', 'arena', 'all'], help="Use mimalloc.")
+        "--use_mimalloc", action='store_true', help="Use mimalloc allocator")
```

Comment:
Are there any internal teams that rely on any of these specific choices? May be the ones who introduced them in the first place? Might want to give them a headsup?

Reply:
I actually checked. Nobody is using it currently.

Reply:
What if someone is building from source ?

Reply:
mimalloc is only available for building from source.

Reply:
Yeah, so what if someone (3P) is building from source after each release ? They might find to their surprise that this option that existed previously doesn't exist anymore.

Reply:
There's no way to know about 3P usage. We can document this in the release notes since this might indeed break existing (if any) 3P workflows.